app/roda_plugins/permit_params.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require 'roda'
+
+module RodaPlugins
+  module PermitParams
+    module InstanceMethods
+      def permit_params(params, allowed_keys)
+        params.slice(*allowed_keys)
+      end
```

Comment:
what are the benefits of this plugin?

Reply:
Reduces the risk of accepting unwanted or malicious parameters by explicitly specifying allowed keys.
I've could use [typecast_params](https://roda.jeremyevans.net/rdoc/classes/Roda/RodaPlugins/TypecastParams.html), but i wanted to keep it simple.